test/freenet/client/async/HealingDecisionSupplierTest.java

```diff
@@ -0,0 +1,44 @@
+package freenet.client.async;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.Arrays;
+
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+public class HealingDecisionSupplierTest {
```

Comment:
This test is missing some stuff for the node’s opennet status but that has already been covered in the other comment. 🙂

Reply:
I added a darknet nest now: always heals in darknet.